jsonpickle/__init__.py

```diff
@@ -53,13 +53,13 @@ def __init__(self, name):
     assert obj.name == result['name'] == 'Awesome'
```

Comment:
This is a nice improvement.  It might make sense to also add from-future imports while we're in here:

```python
from __future__ import absolute_import, division, unicode_literals
```